src/main/java/LongSet.java

```diff
@@ -1,3 +1,5 @@
+import java.util.*;
+
 public class LongSet {
```

Comment:
Good job! The static analysis tool Error Prone reported that the below error [1] used to be here, but you removed it. Check out http://errorprone.info for more information.


[1] ```RecommenderTest/src/main/java/LongSet.java:3: error: cannot find symbol
    Set<Short> s = new HashSet<>();
    ^
  symbol:   class Set
  location: class LongSet```